dist/udb3-angular.css

```diff
@@ -21,6 +21,9 @@ textarea.resize-vertical {
 .dashboard-item .dropdown {
   min-width: 120px;
 }
+.dashboard-item .btn-confirmed2020 {
```

Comment:
Can we make it less time specific. 
We might reuse this feature in the future?

Reply:
👍